subsys/bluetooth/host/Kconfig.l2cap

```diff
@@ -32,6 +32,7 @@ config BT_L2CAP_TX_MTU
 	default 253 if BT_BREDR
 	default 69 if BT_MESH_GATT
 	default 65 if BT_SMP
+	default 64 if BT_AUDIO_UNICAST_SERVER || BT_AUDIO_UNICAST_CLIENT || BT_AUDIO_BROADCAST_SINK || BT_BASS || BT_BASS_CLIENT
```

Comment:
Hmm, actually, since LE Audio mandates security `BT_SMP` is always enabled and this won't really do anything.

Reply:
True, should it be removed then?
The biggest PDU is Paring Public Key used in SC. LE Secure Connections is optional as OOB can be used instead. I wonder if `default 65 if BT_SMP` might be tweaked in the future, so that if LE Audio MTU requirement will not be specified it will result in invalid behavior.

Reply:
Good point. It is true that OOB can be used, and this may possible be reduced. Let's keep as is. 

Kconfig question: If we default to both 65 for SMP and 64 for LE Audio, what will the final value be? The last defined one or the first matched one?

Reply:
I believe the first matched one, as it it does not make sense otherwise.
"Kconfig uses the first default with a satisfied condition, where an empty condition corresponds to if y (is always satisfied)."
https://docs.zephyrproject.org/latest/guides/build/kconfig/setting.html#configuring-invisible-kconfig-symbols